src/operation.ts

```diff
@@ -48,6 +48,23 @@ export class Operation {
             "C-x_r": () => {
                 this.editor.setRMode();
             },
+            "M-y": () => {
+                let that = this;
+                this.editor.yankPop().then(function(result) {
+                    if (result) {
+                        that.editor.setStatusBarMessage("Yank Pop");
+                    } else {
+                        that.editor.setStatusBarMessage("Previous command was not a yank");
+                    }
+                }).catch(function(error) {
+                    console.log(error);
+                });
+            },
+            "backspace": () => {
+                commands.executeCommand("deleteLeft").then(() => {
+                    commands.executeCommand("emacs.exitMarkMode");
```

Comment:
This code should be in `editor.ts` to avoid the import of `vscode.commands` in this file.